templates/conf/fluent.conf.erb

```diff
@@ -82,6 +82,18 @@
   logstash_format "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_FORMAT'] || 'false'}"
   logstash_prefix "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_PREFIX'] || 'logstash'}"
   logstash_prefix_separator "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_PREFIX_SEPARATOR'] || '-'}"
+  reload_connections "#{ENV['FLUENT_OPENSEARCH_RELOAD_CONNECTIONS'] || 'false'}"
+  reconnect_on_error "#{ENV['FLUENT_OPENSEARCH_RECONNECT_ON_ERROR'] || 'true'}"
+  reload_on_failure "#{ENV['FLUENT_OPENSEARCH_RELOAD_ON_FAILURE'] || 'true'}"
```

Comment:
The default value of these 3 items are inverted from plugin's default.
Is there any reason?

Reply:
No, there is no particular reason. I went over the settings once more. I think I copied these from the Elasticsearch template where they are inverted as well. I my newest commit I use the plug-in defaults.